ci/tests/tracing/scenarios/tyk_test-graphql-tracing_200.yml

```diff
@@ -20,9 +20,9 @@ spec:
         - attr:http.url   =   "https://countries.trevorblades.com/"
         - attr:http.method  =  "POST"
     - selector: span[tracetest.span.type="general" name="GraphqlEngine"] span[tracetest.span.type="general"]
-      name: Make sure there are 5 subspans for graphql engine
+      name: Make sure there are 3 subspans for graphql engine
       assertions:
-        - attr:tracetest.selected_spans.count = 5
```

Comment:
an additional assertion you can add is making sure there is only one instance of the validation and normalization spans. That way this change is protected from potential breaks

Reply:
I decided not to add an additional scenario because there's already a test that checks that the graphql engine only has 3 subspans instead of 5